client/src/app/_service-form/service-form.component.html

```diff
@@ -0,0 +1,80 @@
+<form #serviceForm="ngForm" (ngSubmit)="saveService(serviceForm)">
```

Comment:
Maybe we have to refactor in the future the form components, we have a lot of duplicated code in here.

Reply:
should I create an new issue for that?

Reply:
If you want 😄 and if you have time to define one.